Cohere.IntegrationTests/Chat/Chat.feature

```diff
@@ -0,0 +1,37 @@
+@Ignore
```

Comment:
The Integration tests are actually making calls to the Cohere API so they are ignored to prevent pipeline builds failing due to "Too Many Requests" error.